frontend/dispatcher.lua

```diff
@@ -483,7 +479,6 @@ local dispatcher_menu_order = {
     "kopt_zoom_overlap_h",
     "kopt_zoom_overlap_v",
     "kopt_zoom_mode_type",
-    -- "kopt_zoom_range_number", -- can't figure out how this name text func works
```

Comment:
If you figure can out the `name_text func` please leave this in, itdoes work in the bottom menu

Reply:
Thank you, done.